### PR TITLE
Colormaps on surface

### PR DESCRIPTION
**Pull request by KamalakerDadi**
Work which continues upon #1713

### CODE REVIEW

File: doc/plotting/index.rst [hunk 1]
```diff
@@ -231,6 +231,18 @@ Different display modes
 
 ================= =========================================================
 
+Available Colormaps
+===================
+
+Nilearn plotting library ships with a set of extra colormaps, as seen in the
+image below
+
+.. |colormaps| image:: ../auto_examples/01_plotting/images/sphx_glr_plot_colormaps_001.png
```
Comment (GaelVaroquaux) [inline doc/plotting/index.rst]: You need to remove "|colormaps|"

File: examples/01_plotting/plot_colormaps.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Visualize HCP connectome workbench color maps shipped with Nilearn
+which can be used for plotting brain images on surface.
+
+See :ref:`surface-plotting` for surface plotting details.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+from nilearn.plotting.cm import _cmap_d as hcp_cmaps
```
Comment (GaelVaroquaux) [inline examples/01_plotting/plot_colormaps.py]: "as nilearb_cmaps"

File: examples/01_plotting/plot_colormaps.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+###########################################################################
+# Plot color maps
```
Comment (GaelVaroquaux) [inline examples/01_plotting/plot_colormaps.py]: Could you call this "Plot nilearn color maps"

File: examples/01_plotting/plot_colormaps.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+# Initialize the figure
+plt.figure(figsize=(10, 4.2))
+plt.subplots_adjust(top=0.4, bottom=0.05, left=0.01, right=0.99)
+
+for index, cmap in enumerate(hcp_cmaps):
+    plt.subplot(1, nmaps + 1, index + 1)
+    plt.imshow(a, cmap=hcp_cmaps[cmap])
+    plt.axis('off')
+    plt.title(cmap, fontsize=10, va='bottom', rotation=90)
+
```
Comment (GaelVaroquaux) [inline examples/01_plotting/plot_colormaps.py]: Could you add another cell here that plots the maptlotlib color maps (as in the example that I linked)

Comment (KamalakerDadi) [reply examples/01_plotting/plot_colormaps.py]: Matplotlib has lot? Should I pick the ones that are demonstrated in the example link http://www.scipy-lectures.org/intro/matplotlib/auto_examples/options/plot_colormaps.html

Comment (KamalakerDadi) [reply examples/01_plotting/plot_colormaps.py]: I am thinking of adding a cell which displays 4 colormaps on brains ?

**Comment by GaelVaroquaux** [comment]
> I am thinking of adding a cell which displays 4 colormaps on brains ?

Sounds good! Ideally we should use the new neurosynth downloaders, but
they are still not merged :(. Maybe in 20mn?

**Comment by GaelVaroquaux** [comment]
This is awesome, merging!

**Pull request merged**